superset-frontend/src/SqlLab/components/App/index.jsx

```diff
@@ -127,7 +127,7 @@ class App extends React.PureComponent {
     ) {
       this.showLocalStorageUsageWarning(
         this.props.localStorageUsageInKilobytes,
-        this.props.queries?.lenghth || 0,
+        this.props.queries?.length || 0,
```

Comment:
Whoa, interesting catch here. 

CC @justinpark in case this poses any risk... or actually fixes something, since this line was ALWAYS returning 0 in its current implementation.

Reply:
This value only uses for logging so not a huge critical. (Thanks for catching up. I found this issue when I queried the this new logging item first time)

Reply:
This is one of the important reason for typescript!